src/additional-functions/checkout.js

```diff
@@ -4,27 +4,24 @@ import * as mpApi from "../utils/api/monitoringPlansApi";
 export const checkoutAPI = (
   direction,
   configID,
-  selectedConfig,
+  monitorPlanId,
   setCheckout
 ) => {
   const user = JSON.parse(sessionStorage.getItem("cdx_user"));
 
   if (!direction) {
-    mpApi
-      .deleteCheckInMonitoringPlanConfiguration(selectedConfig.id)
+    return mpApi
+      .deleteCheckInMonitoringPlanConfiguration(monitorPlanId)
       .then((res) => {
         console.log(res, "checked back in ");
-        setCheckout(false, configID);
+        if (setCheckout !== undefined) setCheckout(false, configID);
```

Comment:
should be re-written to include {} after if.

Example:
    if (condition) {
      // code here
    }